.github/workflows/benchmark.yml

```
@@ -0,0 +1,78 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Benchmarks
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

```suggestion
    paths: 
      - ".github/workflows/benchmark.yml"
```
Does it make sense to run this on every PR when we can't upload the results and don't display them or anything either?

---

Yes, you are correct; it makes no sense to run this on every PR, as I discovered just now, so I will follow your suggestion to trigger only when changes are made to the benchmark runner file itself.